frontend/src/components/pages/CampCreation/ScheduleSessions/index.tsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import { Box, Text } from "@chakra-ui/react";
+
+const ScheduleSessionsPage = (): JSX.Element => {
```

Comment:
Preference for `JSX.Element` vs. `React.ReactElement`?

Reply:
Looks like we use both in the codebase haha... but I prefer JSX.Element like you've done.